src/main/java/frc/robot/constants/RobotMap.java

```diff
@@ -30,9 +30,13 @@
             public static final int REAR_RIGHT_ANGLE_ENCODER_ID = 10;
         }
 
-        public static class MOTOR_ID {
+        public static class IntakeOpener {
             public static final int INTAKE_OPENER_MOTOR_ID = 3;
```

Comment:
delete INTAKE_ in line 34

Reply:
@Mboesn sould we do that in the intake branch?

Reply:
In theory yes, but just change it here

Reply:
kkk